tests/post_training/test_templates/test_min_max.py

```diff
@@ -166,3 +167,42 @@ def test_get_channel_axes_matmul_torch(self, weight_shape, ref_axes):
         matmul_node = NNCFNode({"metatype": self.matmul_metatype})
         matmul_node.layer_attributes = self.get_matmul_node_attrs(None, None, weight_shape)
         assert self.backend().get_weight_quantization_axes(matmul_node, "dummy") == ref_axes
+
+
+class TemplateTestCommonMinMax(TemplateTestMinMaxAlgorithm):
+    @staticmethod
+    @abstractmethod
+    def get_backend():
+        "Returns backend."
+
+    @staticmethod
+    @abstractmethod
+    def get_no_quantized_ops_graph():
+        "Returns a NNCFGraph with no operations for quantization."
+
+    @pytest.mark.parametrize("run_nums", (2,))
```

Comment:
Looks like it can be hardcoded in test body, no need to add extra parameter for one value

Reply:
done